### ISSUE TITLE
[Windows] Adding an Image as child of ScrollView without size properties crashes the application

### ISSUE THREAD

**Issue by Deepak1799** [dotnet/maui#5898]
### Description

Adding an Image as child of ScrollView without size properties crashes the application and throws the unhandled exception message "Element not found. (0x80070490)".

### Steps to Reproduce

1. Create File -> New -> MaUI App.
2. Add a scroll view control.
3. Add a image to the scroll view (without Height and Width properties) as shown in the code below.

```
    <ScrollView>
                <Image
                Source="dotnet_bot.png"
                SemanticProperties.Description="Cute dot net bot waving hi to you!"/>
    </ScrollView>
```

### Version with bug

Preview 14 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows SDK 10.0.19041

### Did you find any workaround?

_**Note: This is not the workaround, but this can be a hint on how to fix this.**_

In the following ways, the application was not crashed.
1. When setting the height and width property to the image the application is not crashed as shown below.

```
    <ScrollView>
                <Image
                Source="dotnet_bot.png"
                SemanticProperties.Description="Cute dot net bot waving hi to you!"
                WidthRequest="250"
                HeightRequest="310"/>
    </ScrollView>

```

2. Even without Width and Height properties, when using image in any other parent except ScrollView (such as ContentView), does not crashes the app as shown in the following code.

```
    <ContentView>
                <Image
                Source="dotnet_bot.png"
                SemanticProperties.Description="Cute dot net bot waving hi to you!"/>
    </ContentView>
```

### Relevant log output

Element not found. (0x80070490)

**Comment by kristinx0211** [comment]
verified not repro on windows, vs 17.3.0 Preview 1.0 [32406.489.main].
`    <ScrollView>
                <Image
                Source="dotnet_bot.png"
                SemanticProperties.Description="Cute dot net bot waving hi to you!"/>
    </ScrollView>`
build and run succeed without errors.
![image](https://user-images.githubusercontent.com/99457000/162361985-ba0d1b30-433b-4b89-9220-51052cb8b7e8.png)

**Comment by Deepak1799** [comment]
@kristinx0211 , Thank you for the reply. The issue occurred in the latest vs 17.2.0 Preview 2.1 and will wait for the vs 17.3.0 Preview 1.0 release.

But I could see a text "Hello World" in your attached image (as I have shown below). So I suspect you may used the scroll view inside a ViewGroup. 

<img width="633" alt="image" src="https://user-images.githubusercontent.com/65169077/162413069-d20e7db7-9444-45a1-9dc4-7bbf7fa4c4fa.png">

When checked using scroll view inside a ViewGroup (StackLayout), it has worked for me too in the current version. However the app crash issue still occurs if we directly used the Scroll View in the content page.

I have attached the repo project here. Can you please verify this scenario also work in the release?

[ScrollViewCrash.zip](https://github.com/dotnet/maui/files/8450837/ScrollViewCrash.zip)

Thanks.

**Comment by janseris** [comment]
17.2.0 Preview 2.1 
Release Build crash without exception message
Debug Build crash and debugger does not show exception - strange behavior - VS prompts to select a debugger and when a debugger is selected, says that cannot attach debugger because there is one already selected.

https://user-images.githubusercontent.com/64279914/162520675-732f16b8-ae4e-4ce1-be91-47297c8f3776.MP4

**Comment by jsuarezruiz** [comment]
Tested with VS 17.3.0 Preview 1.0 and also in the main branch. In both cases cannot reproduce the issue.
![issue-5898](https://user-images.githubusercontent.com/6755973/164402334-0cb4ba57-4d23-46b3-947f-078c71f17887.gif)

**Comment by Deepak1799** [comment]
Yes, it looks the issue is resolved with VS 17.3.0 Preview 1.0. So closing this issue. Thank you for tracking.
